src/main/java/controllers/UserControllerImpl.java

```diff
@@ -0,0 +1,47 @@
+package controllers;
+
+import dto.UserResponseDto;
+import java.util.List;
+import model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import service.UserMapper;
+import service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserControllerImpl implements UserController {
```

Comment:
for controller, we don't use interfaces usually. /user mapping will decide which class will handle the requests